Authored by:  David Tardon David Tardon 2018-12-30
Commit 333c8a26f231bea26ec3d56245315041bbf5577f

fix build with gcc 4.8

--- a/src/lib/WPXTable.h
+++ b/src/lib/WPXTable.h
@@ -53,7 +53,7 @@
 	~WPXTable();
 	void insertRow();
 	void insertCell(unsigned char colSpan, unsigned char rowSpan, unsigned char borderBits);
-	const WPXTableCell  *getCell(size_t i, size_t j)
+	const WPXTableCell  *getCell(std::size_t i, std::size_t j)
 	{
 		return &(m_tableRows[i])[j];
 	}
